services/core-web/common/components/tailings/BasicInformation.js

```diff
@@ -141,10 +146,10 @@ export const BasicInformation = (props) => {
         id="tsf_operating_status_code"
         name="tsf_operating_status_code"
         label="Operating Status"
-        data={TSF_OPERATING_STATUS_CODE}
+        data={includeClosedStatus}
```

Comment:
I don't like this variable name, it sounds like a boolean.

Reply:
Fair enough, I see your point.  How about `parsedStatusCodeOptions` or something similar?